### ISSUE TITLE
SVG images cannot use text inside

### ISSUE THREAD

**Issue by qarmin** [sharazam/printpdf#184]
Usvg dependency supports embedding text inside svg, but printpdf not

Svg used to test behavior
```
<svg width="100" height="100" xmlns="http://www.w3.org/2000/svg">
  <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />
  <text x="50" y="55" font-size="20" text-anchor="middle" fill="white">A</text>
</svg>
```

![image](https://github.com/user-attachments/assets/c31167f4-b435-4cb5-818a-7ddf4f04f58c)

![image](https://github.com/user-attachments/assets/f5c0016f-61fa-45a1-9dce-9aadac255450)

**Comment by fschutt** [comment]
Yeah, this is a pretty normal problem since PDF handles fonts / text specially. You'd have to first use a SVG library to convert the text to paths.

**Comment by nilscc** [comment]
> You'd have to first use a SVG library to convert the text to paths.

@fschutt Do you know any rust libraries who can do that?
